core/arch/arm/kernel/secure_partition.c

```diff
@@ -349,7 +349,7 @@ static bool sp_handle_svc(struct thread_svc_regs *regs)
 	return false;
 }
 
-static const struct ts_ops sp_ops __rodata_unpaged = {
+static const struct ts_ops sp_ops __rodata_unpaged("sp_ops") = {
```

Comment:
Could you avoid repeating the variable name? (I see a potential difficulty with checkpatch but we may ignore it)

Reply:
Yes, but then we'd have a macro that changes the syntax and that's in this case even worse in my opinion.

Reply:
Fair enough, let's keep this.